.gitignore

```diff
@@ -56,3 +56,5 @@ scripts/_curl
 curl_fuzzer
 curl_fuzzer_seed_corpus.zip
 libstandaloneengine.a
+.vscode/
+*.debugtarget
```

Comment:
Being development environment specific, these two are probably better handled locally in `$GIT_DIR/info/exclude`